Resources/html/views/history/history.js

```diff
@@ -175,9 +175,13 @@ var loadCommit = function(commitObject, currentRef) {
 		$("committerDate").parentNode.style.display = "none";
 	}
 
+    var cveBaseURL = "https://cve.mitre.org/cgi-bin/cvename.cgi?name="
+    var awsBulletinsURL = "https://aws.amazon.com/security/security-bulletins/"
     var textToHTML = function (txt) {
         return (" "+txt+" ")
             .replace(/(https?:\/\/([^\s\.\)\]\<]+|\.[^\s])+)/ig, "<a href=\"$1\">$1</a>")
+            .replace(/(\s)(CVE-\d{4}-\d+)\b/ig, "$1<a href=\""+cveBaseURL+"$2\">$2</a>")
```

Comment:
Would `\b` be better than `\s` for the start of this, as it matches word boundaries? I think it would match the start of a message an you wouldn't need to include it back as $1 in the replace.

Reply:
I initially started with`/\b(CVE-…` however it also matches a CVE that is part of a URL. I then wanted to use negative lookbehind (`/(?<!\S)(CVE-…`) but I had to find out that it's only working in my Google Chrome JS Console and not in WebKit/Safari…

So, I ended up with explicitly capturing whitespace... Any other recommendation more than welcome… ;-)